Editor/AddressableImporter.cs

```diff
@@ -183,7 +217,7 @@ private static void CheckFolders()
             if (filesToImport.Count > 0)
             {
                 Debug.Log($"AddressablesImporter: Found {filesToImport.Count} assets...");
-                OnPostprocessAllAssets(filesToImport.ToArray(), null, null, null);
+                OnPostprocessAllAssets(filesToImport.ToArray(), new string[0], new string[0], new string[0]);
```

Comment:
Oh, will passing null cause any issue?

Reply:
`OnPostprocessAllAssets()` passes an empty array instead of a null if the target does not exist.
This behavior eliminates the null check for each argument.

https://docs.unity3d.com/ScriptReference/AssetPostprocessor.OnPostprocessAllAssets.html

I made the same behavior as this to use the third argument of `OnPostprocessAllAssets()`.

Reply:
If the official example assumes these parameters are not null, then fine.